embedded/tbtree/tbtree.go

```diff
@@ -498,6 +509,20 @@ func OpenWith(path string, nLog, hLog, cLog appendable.Appendable, opts *Options
 		return nil, ErrCorruptedCLog
 	}
 
+	maxKeySize, ok := metadata.GetInt(MetaMaxKeySize)
+	if !ok {
+		maxKeySize = opts.maxKeySize
+	}
+
+	maxValueSize, ok := metadata.GetInt(MetaMaxValueSize)
+	if !ok {
+		maxValueSize = opts.maxValueSize
+	}
+
+	if maxNodeSize < minNodeSize(maxKeySize, maxValueSize) {
+		return nil, fmt.Errorf("%w: max node size is too small for specified max key and max value sizes", ErrIllegalArguments)
```

Comment:
We should add some extra log info here with information on where to find a solution in such case

Reply:
we should describe in the documentation how to address this error. but it should not be associated to a specific documentation site